scripts/kubescape-backlog-bridge/issues.go

```
@@ -0,0 +1,474 @@
+}
+
+// empty reports whether the plan performs no writes at all. Unchanged cluster
+// state MUST produce an empty plan — that is the anti-churn acceptance
+// criterion, expressed as a property the caller can assert.
+//
+// Keyed on the actions alone: a withheld close is by definition not a write,
+// and folding it in here would make a gated run look like it had work to do.
+func (p plan) empty() bool { return len(p.Actions) == 0 }
+
+// zeroWidthSpace breaks a token without removing information from it.
+const zeroWidthSpace = "​"
```

golangci-lint (2.12.2)

[error] ST1018: string literal contains the Unicode format character U+200B, consider using the '\u200b' escape sequence instead

(staticcheck)